src/org/labkey/targetedms/chromlib/AbstractLibTransitionOptimization.java

```diff
@@ -0,0 +1,46 @@
+package org.labkey.targetedms.chromlib;
+
+import java.util.Objects;
+
+public abstract class AbstractLibTransitionOptimization extends AbstractLibEntity
+{
+    protected Integer _transitionId;
```

Comment:
I don't think this should ever be null, correct? If so, please use int instead of Integer